### ISSUE TITLE
Experimental results on HELEN dataset significantly lower than reported

### ISSUE THREAD

**Issue by beefbill** [PRIS-CV/WFEN#7]
Hi, thank you for your great work and for sharing the code and models.

I’m currently attempting to reproduce the results reported in your paper, particularly on face super-resolution tasks. However, I’ve encountered a notable inconsistency when evaluating on the HELEN dataset.

Specifically, the results I obtained are significantly lower than those reported in the paper. This discrepancy appears on HELEN.

I noticed that the HELEN test set used in this paper is consistent with that used in DIC and SPARNet. However, the PSNR values reported in this paper for DIC and SPARNet are significantly lower than those reported in their original papers.

For DIC, this paper reports a PSNR of **26.15 dB**, which is much lower than the **26.61 dB** reported in the original work.

For SPARNet, this paper reports **26.43 dB**, compared to **26.97 dB** in the original paper.

Could you please explain why such large discrepancies in results occur, given that the test data appears to be the same?

**Comment by 24wenjie-li** [comment]
Thanks for your attention. The reason for the low performance of DIC and
SPARNet in the paper compared to the original paper is that DIC and SPARNet
use the full CelebA training set (about 210,000 sheets), so the results
obtained have higher performance. In order to save training time (DIC
training takes about 7 days to train), we describe in the details that we
only used the first 18,000 sheets to replicate the existing method as well
as our method for a fair comparison. So it is normal that less training
data leads to lower performance of DIC and SPARNet.

As for the differences in the HELEN test set, first of all, the pre-trained
model we published is consistent with the paper results. I think it might
be that the way you crop plus align the faces for the training set is
different from ours, as different cropping plus alignment will have some
impact on the performance. You can try to reason our Celeba test set with
the official weights of SPARNet or DIC and get significantly lower results
as well.

bill ***@***.***> 于2025年4月23日周三 22:27写道：

**Comment by beefbill** [comment]
Thanks for your detailed explanation and quick response.  That makes  sense.
